io/disk/Avago_storage_adapter/avago3008.py

```diff
@@ -44,6 +44,7 @@ def setUp(self):
         self.disk = str(self.params.get('disk')).split(" ")
         self.spare = str(self.params.get('spare'))
         self.size = int(self.params.get('size', default='max'))
+        self.tool_location = self.params.get('tool_location')
```

Comment:
User is expected to to give this, and we can't have default values, as if the tool is not present in that location, script will error out.